cobbler/tftpgen.py

```diff
@@ -903,9 +903,10 @@ def build_kernel_options(self, system, profile, distro, image, arch: enums.Archs
                                        % (httpserveraddress, profile.name)
 
             if distro.breed is None or distro.breed == "redhat":
-
-                append_line += " kssendmac"
-                append_line = "%s inst.ks=%s" % (append_line, autoinstall_path)
+                if distro.os_version in ["rhel4", "rhel5", "rhel6", "fedora16"]:
+                    append_line += " kssendmac ks=%s" % autoinstall_path
+                else:
+                    append_line += " inst.ks.sendmac inst.ks=%s" % autoinstall_path
```

Comment:
I expect that this is equivalent to `inst.kssendmac` from https://github.com/cobbler/cobbler/pull/3198/files ?

Reply:
That was changed in the second PR (https://github.com/cobbler/cobbler/pull/3335/files#diff-1637482678b3b5bc9ef6ec9b6b25f9743b9d25783ca5ee7e78f9d0964809eb88) listed in the description. The first implementation was not working for older distros.